owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewAudioFragment.kt

```diff
@@ -342,6 +359,15 @@ class PreviewAudioFragment : FileFragment() {
         }
     }
 
+    private fun showSnackbarMessage(){
+        val snackbarString = if (file.isAvailableOffline){
+            R.string.confirmation_unset_available_offline
+        } else {
+            R.string.confirmation_set_available_offline
+        }
```

Comment:
Use these 2 messages in `FileDetailsFragment` as well

Reply:
I've checked it and the logic in `FileDetailsFragment` is different, so it's not applicable in this case.